modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -5,178 +5,172 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { Selections } from '@ephox/darwin';
-import { Fun } from '@ephox/katamari';
+import { Arr, Fun } from '@ephox/katamari';
 import { SugarNode } from '@ephox/sugar';
 
 import Editor from 'tinymce/core/api/Editor';
 import { Menu } from 'tinymce/core/api/ui/Ui';
 
+import * as FakeClipboard from '../api/Clipboard';
 import * as Options from '../api/Options';
-import { Clipboard } from '../core/Clipboard';
 import { SelectionTargets, LockedDisable } from '../selection/SelectionTargets';
 import { verticalAlignValues } from './CellAlignValues';
 import { applyTableCellStyle, changeColumnHeader, changeRowHeader, filterNoneItem, buildColorMenu, buildMenuItems } from './UiUtils';
 
-const addMenuItems = (editor: Editor, selections: Selections, selectionTargets: SelectionTargets, clipboard: Clipboard): void => {
+interface AddMenuSpec {
+  text: string;
+  command: string;
+  icon?: string;
+  onSetup: (api: Menu.MenuItemInstanceApi) => () => void;
+}
+
+const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets): void => {
   const cmd = (command: string) => () => editor.execCommand(command);
 
+  // TODO: TINY-8172 Unwind this when an alternative solution is found
+  const addMenuIfRegistered = (name: string, spec: AddMenuSpec) => {
+    if (editor.editorCommands.queryCommandSupported(spec.command)) {
```

Comment:
```suggestion
    if (editor.queryCommandSupported(spec.command)) {
```
Might as well use the alias instead here.

Reply:
Thanks. Have fixed all of these in Buttons.ts and MenuItems.ts